tutorials/migration.md

```diff
@@ -481,32 +481,33 @@ A Thing represents a configurable device/system/unit, which provides different f
 Each Channel corresponds exactly to one binding configuration string (stuff in { }) in openHAB 1.x.
 
 Let's look at a concrete example. 
-The [Yahoo Weather Binding]({{base}}/addons/bindings/yahooweather/readme.html) supports exactly one Thing which takes two parameters: a WOEID location and unit.
+The [Network Binding](addons/bindings/network) (the impoved binding to the OH1 Network Health binding) supports exactly one Thing which takes one parameter: the IP number of the device on the network.
 
-Thus, as described in the Binding's readme one would manually define a Thing in a .things file (located in conf/things) with the line:
+Thus, as described in the Binding's readme one would manually define a Thing in a .things file (located in conf/things) with the line (this can also be achieved through the PaperUI:
```

Comment:
```suggestion
Thus, as described in the Binding's readme one would manually define a Thing in a .things file (located in conf/things) with the line (this can also be achieved through the PaperUI):
```

Reply:
Thanks